src/docs/cookbook/maintenance/error-reporting.md

```diff
@@ -193,3 +117,5 @@ see the [Crashy][] example app.
 [Sentry]: https://sentry.io/welcome/
 [`sentry`]: {{site.pub-pkg}}/sentry
 [`Zone`]: {{site.api}}/flutter/dart-async/Zone-class.html
```

Comment:
I think this could be removed as the SDK hooks up automatically

Reply:
Good point these are links and many of them are no longer needed